doc/getting-started/apl-nuc.rst

```diff
@@ -91,6 +91,9 @@ complete this setup.
 #. After installation is complete, boot into Clear Linux OS, login as
    **clear**, and set a password.
 
+#. If you want to set up ACRN hypervisor, Service OS or Guest OS automatically, we recommend you follow
```

Comment:
@dbkinder could you suggest some text to make it clearer that we have two methods described in this doc. One that is more manual (and more complex) and this new one based on the script?

I'm also starting to think that renaming the script may be a good idea, something like acrn_quick_setup.sh, what do you think?

Reply:
Maybe:
```
#. The remaining instructions below provide detailed instructions on setting
   up the ACRN Hypervisor, Service OS, and Guest OS.  We also provide an
   automated script that does all these steps for you, so you can skip these
   manual steps.  See the `quick-setup-guide`_ section below to use the
   automated setup script.
```

Actually, my preference would be to put the automated script instructions
immediately below, and then show the "hard way" after that.

Reply:
Also, we need to rewrite the Clear Linux steps listed above this step.  The whole user interface for installing Clear Linux has changed and the options aren't named the same or are in different orders.

Reply:
``gsg_quick_setup.sh`` script now is changed to ``acrn_quick_setup.sh``
And move the script instructions immediately below to the 'Set up a Clear Linux Operating System', the installing steps is also changed based on 28960.